app/src/ui/about/about.tsx

```diff
@@ -232,7 +237,7 @@ export class About extends React.Component<IAboutProps, IAboutState> {
           </Row>
           <h2>{name}</h2>
           <p className='no-padding'>
-            Version {version} ({releaseNotesLink})
+            Version <span title='Click to copy' className='version-text' onClick={this.onClickVersion}>{version} <Octicon symbol={OcticonSymbol.clippy} /></span> ({releaseNotesLink})
```

Comment:
@shiftkey So something like this?

> Version [0.6.0] ([release notes](https://desktop.github.com/release-notes/))

[0.6.0]: #discussion_r121018222 "Click to copy"